backend/src/journey/service/mapper/journeyMapper.service.ts

```diff
@@ -92,7 +92,7 @@ export class JourneyMapperService {
       // 5a) Check stop sequence locations information are present.
       // It is checked before if stop sequence is present (in DetailsMapper).
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      && this.journeyLocationMapper.checkVrrJourneyLocationsIntegrity(vrrLeg.stopSequence!)
+      && this.journeyLocationMapper.checkVrrJourneyLocationIntegrity(vrrLeg.stopSequence!)
```

Comment:
Interesting that this seems to be causing no compile/build error (maybe because of the exclamation mark?). Also a good example of the limitation of the unit tests I added. As the other mappers are always mocked things like wrong parameters aren´t seen until running the application. But E2E tests should fail, I guess, because no journeys are left if this condition always fails.

Reply:
Its not the exclamation 
```ts
[property: string]: any;
```
at VrrAPITypes.ts line 65. This essentially makes every Object-like type assignable to JourneyLocationElement.